src/pal/src/misc/perftrace.cpp

```diff
@@ -33,7 +33,7 @@ Module Name:
 #include <string.h>
 #include <errno.h>
 
-#define snprintf _snprintf
+#define snprintf _snprintf_s
```

Comment:
>snprintf [](start = 8, length = 8)

Is there going to be compile error if PAL_PERF is ever enabled? I guess it does not matter since you are planning to delete this file?